data/remnant/remnant outfits.txt

```diff
@@ -529,6 +542,10 @@ outfit "Smelter-Class Thruster"
 	"thrust" 76.8
 	"thrusting energy" 10.10
 	"thrusting heat" 10.2
+	"reverse thrust" 46.08
+	"reverse thrusting energy" 6.06
+	"reverse thrusting heat" 1.5
+	"slowing resistance" 2.0
```

Comment:
We just got through how resistance values like this for ionization result in being able to completely ignore large amounts of ionization. You should REALLY start using much smaller resistance values whenever you add them to stuff. (This value means that a single Detainer's slowing will be completely removed after only 35 frames from when the first shot in a burst hits. That's just over half a second. And if you look at having TWELEVE Inhibitors, this is enough resistance to bleed off the slowing in 3 frames.)

Whenever using a resistance, you should be looking at the values of the weapons that use the related status effect. For example, look at the Inhibitor and Detainer for slowing damage. The Inhibitor deals 0.5 damage while the Detainer deals 40. A resistance of 2 means a single Inhibitor never does anything to you, while a Detainer only mildly inconveniences you for 20 frames, since resistance is done every frame.

I'm still not on board with slowing resistance on Remnant engines anyway, though.

Reply:
Reduced the values to 1/5 of this. That being said, a large engine from the faction that is a) devoted to running away, and b) does have an understanding of how slowing effects work? To me, that makes perfect sense that such an engine would be able to shrug off a single inhibitor shot without effect. They probably get  hit by them all the time during training exercises and combat training. Makes a lot of sense that they'd build their stuff to be resistant to their own stuff.

Anyway, reduced that from 2 down to 0.4.